packages/babel-compiler/babel.js

```diff
@@ -1,52 +1,30 @@
-Babel = Npm.require('babel-core');
+var meteorBabel = Npm.require('meteor-babel');
 
 // See README.md in this directory for more information.
 
-Babel.transformMeteor = function (code, extraOptions) {
-  extraOptions = _.extend({}, extraOptions); // clone
+Babel = {
+  transformMeteor: function (code, extraOptions) {
+    // See https://github.com/meteor/babel/blob/master/options.js if you
+    // want to know what these default options are.
+    var options = meteorBabel.getDefaultOptions();
+    delete options.sourceMap;
```

Comment:
The default options have `sourceMap: "inline"`, which breaks the tests in this repository.